apps/src/sites/studio/pages/levels/_curriculum_reference.js

```diff
@@ -0,0 +1,33 @@
+window.curriculumReference = {
+  onload: onload
+};
+
+/**
+ * Called on load of our iframe. This hides the spinner, shows the iframe, and
+ * sets up a method to keep our iframe correctly sized.
+ */
+function onload() {
+  const iframe = document.getElementById('curriculum-reference');
+  // For some reaosn onload is called multiple times in Chrome. We only want to
+  // start our interval once.
+  if (iframe.style.display !== 'none') {
+    return;
+  }
+
+  // Make iframe visible, hide spinner
+  iframe.style.display = '';
+  document.getElementById('iframe-loading').style.display = 'none';
+
+  // Our iframe can resize for various reasons (initial load, images loaded on
+  // iframe page, window resized). We want to keep the size of the iframe in
+  // sync with the contents of the firame.
+  setInterval(() => {
+    resizeIFrame(iframe);
+  }, 200);
```

Comment:
Does setting the iframe height to `100%` or [`calc(100% - 80px)`](http://www.w3.org/TR/css3-values/#calc-notation) work?  If not, can we bind to specific window events like "resize", or "load" on the iframe document [using capture](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters)?